### ISSUE TITLE
Issues with aasserver.json for DotAAS V3 

### ISSUE THREAD

**Issue by aaronzi** [eclipse-basyx/basyx-databridge#265]
This is mainly a documentation issue I would say.

## BaSyx wiki:

First of all, in the [BaSyx wiki for the DataBridge](https://wiki.eclipse.org/BaSyx_/_Documentation_/_Components_/_DataBridge_/_Features_/_AAS_Endpoint) is an example configuration for the AAS Endpoint when using DotAAS V3:

```json
[
    {
        "uniqueId": "ConnectedSubmodel/ConnectedPropertyA",
        "submodelEndpoint": "http://localhost:4001/shells/TestUpdatedDeviceAAS/aas/submodels/ConnectedSubmodel/submodel",
        "idShortPath": "ConnectedPropertyA",
        "api": "DotAAS-V3"
    }
]
```

Here, the main issue is the **submodelEndpoint** which should not be the endpoint of the AAS Repository but rather the Submodel Repository.

## DataBridge GitHub repository:

Here, an example for the [aasserver.json](https://github.com/eclipse-basyx/basyx-databridge/blob/main/databridge.examples/databridge.examples.dot-aas-v3-api/src/main/resources/aasserver.json) is provided for use with DotAAS V3:

```json
[
    {
        "uniqueId": "ConnectedTestSubmodel/DotAASV3ConformantApiProperty",
        "submodelEndpoint": "http://localhost:4001/submodels/submodelId/submodel",
        "idShortPath": "DotAASV3ConformantApiProperty",
        "api": "DotAAS-V3"
    }
]
```

This example has the **submodelEnpoint** almost right. Here, the suffix `/submodel` has to be removed from the endpoint.

Also, the **submodelId** has to be base64 URL encoded. @mdanish98 said this was implemented but it definitely does not work as of right now. Personally I would say to not implement this as a feature for the DataBridge but rather let the user encode the ID. this way the user does not have to change the endpoint URL when retrieving it from for example the AAS Web UI or the Submodel Repositories Swagger.

Another issue can be seen in the **idShortPath** documentation. The AAS V3 spec states that the elements in the path should be dot-separated. This is currently not done by the DataBridge itself. I would also personally not implement this in the DataBridge but also let the user dot-seperate the elements in the path instead of translating "/" to ".". My argument here is the same as with the SubmodelId encoding.

Here is an example _aasserver.json_ configuration for DotAAS V3 that is currently working:

```json
[
    {
        "uniqueId": "ConnectedTestSubmodel/DotAASV3ConformantApiProperty",
        "submodelEndpoint": "http://localhost:4001/submodels/{base64_encoded_submodel_identifier}",
        "idShortPath": "V3ConformantSubmodelElementCollection.DotAASV3ConformantApiProperty",
        "api": "DotAAS-V3"
    }
]
```

**Comment by mdanish98** [comment]
Hi @aaronzi 

Thanks for raising this issue.

We will take a deeper look on the documentation part and will correct it.

Regarding the part **"Also, the submodelId has to be base64 URL encoded. @mdanish98 said this was implemented but it definitely does not work as of right now."** 
I think there might be some communication gap, as I wrote that **"yes, we have to provide the encoded IDs for SM and AAS because DataBridge doesn’t do that currently.".**
It means that we have to provide the encoded IDs in the configuration because DataBridge does not encode the provided IDs.
